b2sdk/sync/folder.py

```diff
@@ -233,10 +232,7 @@ def _walk_relative_paths(self, local_dir, b2_dir, reporter, policies_manager):
 
             if os.path.isdir(local_path):
                 name += '/'
-                if policies_manager.should_exclude_directory(b2_path):
-                    continue
-            else:
-                if policies_manager.should_exclude_file(b2_path):
+                if policies_manager.should_exclude_local_directory(b2_path):
```

Comment:
it seems that `b2_path` and `b2_dir` names are misleading - those are actually relative paths - `local_dir` is path on filesystem (it might not be absolute path) - and `b2_dir` is path relative to `self.root` - can we at least change the `b2_*` names so it would not be confusing?